packages/lesswrong/components/form-components/FormComponentMultiSelect.tsx

```diff
@@ -35,7 +37,7 @@ const FormComponentMultiSelect = ({ value, classes, placeholder, options, path }
         return <em className={classes.placeholder}>{placeholder}</em>
       }
       // if any options are selected, display them separated by commas
-      return selected.map(s => options.find(option => option.value === s)?.label).join(', ')
+      return selected.map(s => options.find(option => option.value === s)?.label).join(separator || ', ')
```

Comment:
Since the options for this field are quite long, I wanted to have a newline between each, like so:

<img width="806" alt="Screen Shot 2022-05-16 at 4 18 31 PM" src="https://user-images.githubusercontent.com/9057804/168675684-900b96ea-427d-4d86-85c8-e2da651b65ee.png">